### PR TITLE
Adding snapshot card vm overview

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

> Add a brief description

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabSnapshots/utils/snapshotStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  InProgressIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+
+export const snapshotStatuses = {
+  Succeeded: 'Stopped',
```
Comment (avivtur) [inline src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabSnapshots/utils/snapshotStatus.tsx:11]: lol fix this please

Comment (metalice) [reply src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabSnapshots/utils/snapshotStatus.tsx]: lol

**Pull request merged**